src/utils/hasProp.spec.js

```diff
@@ -0,0 +1,12 @@
+const { expect } = require('chai');
+const { hasProp } = require("./index");
+
+describe('Utils - hasProp', function suite() {
+  it('should correctly handle property detection', function () {
```

Comment:
Separate into different specs

Reply:
What do you mean with different specs ? 
In all case, will be done in separate PR, but please tell me more about this so I can take note

Reply:
I meant when we test different use cases we usually desribe them in different `it` specs